go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/blocky/nitrite
 
-go 1.19
+go 1.21.11
```

Comment:
Since this project is for us, I'm updating it to the go version used in github actions across our projects.